packages/peregrine/lib/talons/CheckoutPage/useCheckoutPage.js

```diff
@@ -159,7 +156,7 @@ export const useCheckoutPage = props => {
         hasError: !!placeOrderError,
         isCartEmpty: !(checkoutData && checkoutData.cart.total_quantity),
         isGuestCheckout: !isSignedIn,
-        isLoading: checkoutLoading,
+        isLoading: !checkoutCalled || (checkoutCalled && checkoutLoading),
```

Comment:
There's a moment in render where we aren't loading but also haven't yet called the query. We want to wait to render anything until we've loaded the step at a minimum.